diff -Naur a/usr/bin/apt-mirror b/usr/bin/apt-mirror
--- a/usr/bin/apt-mirror	2019-11-27 12:22:23.011103450 -0700
+++ b/usr/bin/apt-mirror	2019-11-27 12:22:58.535105219 -0700
@@ -520,7 +520,7 @@
                 if ( @parts == 3 )
                 {
                     my ( $sha1, $size, $filename ) = @parts;
-                    if ( $filename =~ m{^$component/i18n/Translation-[^./]*\.bz2$} )
+                    if ( $filename =~ m{^$component/i18n/Translation-[^./]*\.(gz|bz2|xz)$} )
                     {
                         add_url_to_download( $dist_uri . $filename, $size );
                     }
